[ML] MlJobSnapshotUpgradeIT.testSnapshotUpgrader fails in 7.series upgrading from 6.series

**Build scan**:
 https://gradle-enterprise.elastic.co/s/xmgtcufihtnde/tests/:x-pack:qa:rolling-upgrade:v6.8.14%23upgradedClusterTest/org.elasticsearch.upgrades.MlJobSnapshotUpgradeIT/testSnapshotUpgrader#1
**Repro line**:
```
./gradlew ':x-pack:qa:rolling-upgrade:v6.8.14#upgradedClusterTest' -Dtests.class="org.elasticsearch.upgrades.MlJobSnapshotUpgradeIT" -Dtests.method="testSnapshotUpgrader" -Dtests.seed=696247791F873496 -Dtests.security.manager=true -Dtests.bwc=true -Dtests.locale=mt -Dtests.timezone=Africa/Mogadishu -Druntime.java=11
```
```
./gradlew ':x-pack:qa:rolling-upgrade:v6.8.14#upgradedClusterTest' -Dtests.class="org.elasticsearch.upgrades.MlJobSnapshotUpgradeIT" -Dtests.method="testSnapshotUpgrader" -Dtests.seed=E4C7C7A0B9EBF14A -Dtests.security.manager=true -Dtests.bwc=true -Dtests.locale=ca-ES -Dtests.timezone=Africa/Bamako -Druntime.java=11
```

**Reproduces locally?**:
?
**Applicable branches**:
7.x, 7.11
**Failure history**:
Two recent failures
**Failure excerpt**:

```
org.elasticsearch.upgrades.MlJobSnapshotUpgradeIT > testSnapshotUpgrader FAILED
10:43:49     ElasticsearchStatusException[Elasticsearch exception [type=exception, reason=Unexpected state [failed] while waiting for to be assigned to a node; recorded reason [[ml-snapshots-upgrade-job] autodetect process stopped unexpectedly: SubLevel mismatch in restore at b
10:43:49     Failed to restore CONTROLLER_OLD_TAG, got 1
10:43:49     Fatal error: 'si_signo 11, si_code: 1, si_errno: 0, address: 0x7ffb03edaf3e, library: /dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/distribution/archives/linux-tar/build/install/elasticsearch-7.11.0-SNAPSHOT/modules/x-pack-ml/platform/linux-x86_64/bin/../lib/libMlMaths.so, base: 0x7ffb038a2000, normalized address: 0x638f3e', version: 7.11.0-SNAPSHOT (build 9a819d67d56a96)
10:43:49     ]]]
10:43:49         at __randomizedtesting.SeedInfo.seed([E4C7C7A0B9EBF14A:F8E4DECFBE9E5AC4]:0)
10:43:49         at org.elasticsearch.rest.BytesRestResponse.errorFromXContent(BytesRestResponse.java:187)
10:43:49         at org.elasticsearch.client.RestHighLevelClient.parseEntity(RestHighLevelClient.java:1911)
10:43:49         at org.elasticsearch.client.RestHighLevelClient.parseResponseException(RestHighLevelClient.java:1888)
10:43:49         at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1645)
10:43:49         at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1617)
10:43:49         at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1584)
10:43:49         at org.elasticsearch.client.MachineLearningClient.upgradeJobSnapshot(MachineLearningClient.java:1196)
10:43:49         at org.elasticsearch.upgrades.MlJobSnapshotUpgradeIT.testSnapshotUpgrade(MlJobSnapshotUpgradeIT.java:133)
10:43:49         at org.elasticsearch.upgrades.MlJobSnapshotUpgradeIT.testSnapshotUpgrader(MlJobSnapshotUpgradeIT.java:112)
```

Logs from the 6.8.14 categorization job being opened:

```
[2021-01-19T15:40:42,864][DEBUG][o.e.x.m.j.p.JobResultsProvider] [v6.8.14-1] Found 0 hits for [[.ml-anomalies-ml-snapshots-upgrade-job]]
[2021-01-19T15:40:42,865][DEBUG][o.e.x.m.j.p.JobResultsProvider] [v6.8.14-1] Found 0 hits for [[.ml-anomalies-ml-snapshots-upgrade-job]]
[2021-01-19T15:40:42,865][DEBUG][o.e.x.m.j.p.JobResultsProvider] [v6.8.14-1] Found 0 hits for [[.ml-anomalies-ml-snapshots-upgrade-job]]
[2021-01-19T15:40:42,865][DEBUG][o.e.x.m.j.p.JobResultsProvider] [v6.8.14-1] Found 0 hits for [[.ml-state, .ml-state-write]]
[2021-01-19T15:40:42,874][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [v6.8.14-1] [ml-snapshots-upgrade-job] Loading model snapshot [N/A], job latest_record_timestamp [N/A]
[2021-01-19T15:40:42,886][DEBUG][o.e.x.m.j.p.a.NativeAutodetectProcessFactory] [v6.8.14-1] [ml-snapshots-upgrade-job] Periodic operations staggered by 3084 seconds
[2021-01-19T15:40:42,891][DEBUG][o.e.x.m.j.p.a.NativeAutodetectProcessFactory] [v6.8.14-1] FieldConfig:
detector.0.clause = mean(value) partitionfield=series categorizationfield=text
detector.1.clause = count by mlcategory categorizationfield=text

[2021-01-19T15:40:42,891][DEBUG][o.e.x.m.p.NativeController] [v6.8.14-1] Starting process with command: [./autodetect, --jobid=ml-snapshots-upgrade-job, --bucketspan=3600, --lengthEncodedInput, --maxAnomalyRecords=500, --timefield=time, --persistInterval=13884, --maxQuantileInterval=24684, --limitconfig=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/limitconfig330030665467624222.conf, --fieldconfig=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/fieldconfig9194247967292977084.conf, --logPipe=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job_log_509269, --input=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job_input_509269, --inputIsPipe, --output=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job_output_509269, --outputIsPipe, --persist=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job_persist_509269, --persistIsPipe]
[2021-01-19T15:40:42,891][TRACE][o.e.x.m.n.Auditor        ] [v6.8.14-1] Successfully persisted audit_message
[2021-01-19T15:40:43,190][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [controller/509582] [CDetachedProcessSpawner.cc@291] Spawned './autodetect' with PID 511601
[2021-01-19T15:40:43,264][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CLogger.cc@363] uname -a: Linux elasticsearch-ci-immutable-debian-10-1611066880089981659 4.19.0-13-cloud-amd64 #1 SMP Debian 4.19.160-2 (2020-11-28) x86_64
[2021-01-19T15:40:43,265][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CLogger.cc@276] Logger is logging to named pipe /dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job_log_509269
[2021-01-19T15:40:43,265][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [Main.cc@117] autodetect (64 bit): Version 6.8.14-SNAPSHOT (Build 62d07d5ce1e171) Copyright (c) 2021 Elasticsearch BV
[2021-01-19T15:40:43,265][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CProcessPriority_Linux.cc@33] Successfully increased OOM killer adjustment via /proc/self/oom_score_adj
[2021-01-19T15:40:43,265][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CSystemCallFilter_Linux.cc@122] Seccomp BPF filters available
[2021-01-19T15:40:43,265][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CSystemCallFilter_Linux.cc@148] Seccomp BPF installed
[2021-01-19T15:40:43,265][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CLimits.h@108] Using default value (10000) for unspecified setting autoconfig.events
[2021-01-19T15:40:43,265][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CLimits.h@108] Using default value (1000000) for unspecified setting anomaly.maxtimebuckets
[2021-01-19T15:40:43,265][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CLimits.h@108] Using default value (3.5) for unspecified setting results.unusualprobabilitythreshold
[2021-01-19T15:40:43,266][INFO ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CResourceMonitor.cc@68] Setting model memory limit to 1024 MB
[2021-01-19T15:40:43,266][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CLengthEncodedInputParser.cc@44] Length encoded input parser input is not connected to stdin
[2021-01-19T15:40:43,266][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldConfig.cc@201] Reading config file /dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/fieldconfig9194247967292977084.conf
[2021-01-19T15:40:43,266][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CWordDictionary.cc@109] Populating word dictionary from file /dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/distribution/bwc/maintenance/build/.transforms/65d663e6451621276178ecd96463b143/elasticsearch-6.8.14-SNAPSHOT.zip/elasticsearch-6.8.14-SNAPSHOT/modules/x-pack-ml/platform/linux-x86_64/bin/../resources/ml-en.dict
[2021-01-19T15:40:43,299][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CWordDictionary.cc@141] Populated word dictionary with 76782 words
[2021-01-19T15:40:43,299][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@60] Configuring categorization filtering
[2021-01-19T15:40:43,299][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [Main.cc@249] Applying the categorization typer for anomaly detection
[2021-01-19T15:40:43,299][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CCmdSkeleton.cc@30] No restoration source specified - will not attempt to restore state
[2021-01-19T15:40:43,307][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [v6.8.14-1] Successfully set job state to [opened] for job [ml-snapshots-upgrade-job]
[2021-01-19T15:40:43,410][DEBUG][o.e.x.m.j.p.a.AutodetectProcessManager] [v6.8.14-1] Flushing job ml-snapshots-upgrade-job
[2021-01-19T15:40:43,413][DEBUG][o.e.x.m.j.p.a.AutodetectCommunicator] [v6.8.14-1] [ml-snapshots-upgrade-job] waiting for flush
[2021-01-19T15:40:43,414][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CAnomalyDetector.cc@119] CAnomalyDetector(): count by count for '', first time = 1491004800, bucketLength = 3600, m_LastBucketEndTime = 1491004800
[2021-01-19T15:40:43,414][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CAnomalyDetector.cc@119] CAnomalyDetector(): mean value partition=series/foo for 'foo', first time = 1491004800, bucketLength = 3600, m_LastBucketEndTime = 1491004800
[2021-01-19T15:40:43,414][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CAnomalyDetector.cc@119] CAnomalyDetector(): count by mlcategory for '', first time = 1491004800, bucketLength = 3600, m_LastBucketEndTime = 1491004800
[2021-01-19T15:40:43,414][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491006600
[2021-01-19T15:40:43,415][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491010200
[2021-01-19T15:40:43,416][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491013800
[2021-01-19T15:40:43,417][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491017400
[2021-01-19T15:40:43,417][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491021000
[2021-01-19T15:40:43,418][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,100.0,,foo,1491024600
[2021-01-19T15:40:43,419][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491028200
[2021-01-19T15:40:43,420][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491031800
[2021-01-19T15:40:43,421][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491035400
[2021-01-19T15:40:43,422][WARN ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@145] Assigning type -1 to record with blank text field:
.,...,value,text,series,time
,,10.0,,foo,1491039000
[2021-01-19T15:40:43,425][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: to index .ml-anomalies-.write-ml-snapshots-upgrade-job with ID [ml-snapshots-upgrade-job_category_definition_1]
[2021-01-19T15:40:43,442][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: to index .ml-anomalies-.write-ml-snapshots-upgrade-job with ID [ml-snapshots-upgrade-job_category_definition_2]
[2021-01-19T15:40:43,456][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Deleting interim results
[2021-01-19T15:40:43,497][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491004800000_3600]
[2021-01-19T15:40:43,498][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,512][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 1 parsed from output
[2021-01-19T15:40:43,512][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Parsed ModelSizeStats: 64880 / 5 / 0 / 3 / 0 / ok
[2021-01-19T15:40:43,512][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] Persisting model size stats, for size 64880
[2021-01-19T15:40:43,512][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: to index .ml-anomalies-.write-ml-snapshots-upgrade-job with ID [ml-snapshots-upgrade-job_model_size_stats_1611070843000]
[2021-01-19T15:40:43,524][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491008400000_3600]
[2021-01-19T15:40:43,524][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,535][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 2 parsed from output
[2021-01-19T15:40:43,536][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Parsed ModelSizeStats: 75234 / 6 / 0 / 3 / 0 / ok
[2021-01-19T15:40:43,536][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] Persisting model size stats, for size 75234
[2021-01-19T15:40:43,536][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: to index .ml-anomalies-.write-ml-snapshots-upgrade-job with ID [ml-snapshots-upgrade-job_model_size_stats_1611070843000]
[2021-01-19T15:40:43,547][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: to index .ml-anomalies-.write-ml-snapshots-upgrade-job with ID [ml-snapshots-upgrade-job_category_definition_2]
[2021-01-19T15:40:43,558][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491012000000_3600]
[2021-01-19T15:40:43,558][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,567][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 3 parsed from output
[2021-01-19T15:40:43,567][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491015600000_3600]
[2021-01-19T15:40:43,567][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,580][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 4 parsed from output
[2021-01-19T15:40:43,580][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491019200000_3600]
[2021-01-19T15:40:43,581][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,593][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 5 parsed from output
[2021-01-19T15:40:43,594][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491022800000_3600]
[2021-01-19T15:40:43,594][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,605][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 6 parsed from output
[2021-01-19T15:40:43,605][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491026400000_3600]
[2021-01-19T15:40:43,605][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,618][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 7 parsed from output
[2021-01-19T15:40:43,618][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Parsed ModelSizeStats: 73992 / 6 / 0 / 3 / 0 / ok
[2021-01-19T15:40:43,619][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] Persisting model size stats, for size 73992
[2021-01-19T15:40:43,619][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: to index .ml-anomalies-.write-ml-snapshots-upgrade-job with ID [ml-snapshots-upgrade-job_model_size_stats_1611070843000]
[2021-01-19T15:40:43,632][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491030000000_3600]
[2021-01-19T15:40:43,632][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,645][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 8 parsed from output
[2021-01-19T15:40:43,645][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: index bucket to index [.ml-anomalies-.write-ml-snapshots-upgrade-job] with ID [ml-snapshots-upgrade-job_bucket_1491033600000_3600]
[2021-01-19T15:40:43,645][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: bulk request with 1 actions
[2021-01-19T15:40:43,661][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Bucket number 9 parsed from output
[2021-01-19T15:40:43,661][DEBUG][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Flush acknowledgement parsed from output for ID 1
[2021-01-19T15:40:43,661][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: refresh index .ml-anomalies-ml-snapshots-upgrade-job
[2021-01-19T15:40:43,690][DEBUG][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Flush acknowledgement sent to listener for ID 1
[2021-01-19T15:40:43,690][DEBUG][o.e.x.m.j.p.a.AutodetectCommunicator] [v6.8.14-1] [ml-snapshots-upgrade-job] Initial flush completed, waiting until renormalizer is idle.
[2021-01-19T15:40:43,690][DEBUG][o.e.x.m.j.p.a.AutodetectCommunicator] [v6.8.14-1] [ml-snapshots-upgrade-job] Flush completed
[2021-01-19T15:40:43,737][DEBUG][o.e.x.m.j.p.a.AutodetectProcessManager] [v6.8.14-1] Attempting to close job [ml-snapshots-upgrade-job], because [close job (api)]
[2021-01-19T15:40:43,738][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [v6.8.14-1] Closing job [ml-snapshots-upgrade-job], because [close job (api)]
[2021-01-19T15:40:43,739][INFO ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CCmdSkeleton.cc@45] Handled 20 records
[2021-01-19T15:40:43,739][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CJsonOutputWriter.cc@813] Wrote quantiles state at 1611070843
[2021-01-19T15:40:43,739][INFO ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CAnomalyJob.cc@1366] Pruning all models
[2021-01-19T15:40:43,740][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CAnomalyJob.cc@1111] Persisted state for 'count by count'
[2021-01-19T15:40:43,742][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CAnomalyJob.cc@1111] Persisted state for 'count by mlcategory'
[2021-01-19T15:40:43,743][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CAnomalyJob.cc@1111] Persisted state for 'mean value partition=series/foo'
[2021-01-19T15:40:43,745][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CModelSnapshotJsonWriter.cc@78] Wrote model snapshot report with ID 1611070843 for: State persisted due to job close at 2021-01-19T15:40:43+0000, latest final results at 1491033600
[2021-01-19T15:40:43,745][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CFieldDataTyper.cc@333] Persist typer state
[2021-01-19T15:40:43,745][DEBUG][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Parsed Quantiles with timestamp 2017-04-01T08:00:00.000+0000
[2021-01-19T15:40:43,746][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: to index .ml-state-write with ID [ml-snapshots-upgrade-job_quantiles]
[2021-01-19T15:40:43,747][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [Main.cc@279] [{"name":"E_NumberNewPeopleNotAllowed","description":"Number of new people not allowed","value":0}
,{"name":"E_NumberNewPeople","description":"Number of new people created","value":6}
,{"name":"E_NumberNewPeopleRecycled","description":"Number of new people recycled into existing space","value":0}
,{"name":"E_NumberApiRecordsHandled","description":"Number of records successfully ingested into the engine API","value":20}
,{"name":"E_MemoryUsage","description":"The estimated memory currently used by the engine and models","value":37036}
,{"name":"E_NumberMemoryUsageChecks","description":"Number of times a model memory usage check has been carried out","value":8}
,{"name":"E_NumberMemoryUsageEstimates","description":"Number of times a partial memory usage estimate has been carried out","value":36}
,{"name":"E_NumberRecordsNoTimeField","description":"Number of records that didn't contain a Time field","value":0}
,{"name":"E_NumberTimeFieldConversionErrors","description":"Number of records where the format of the time field could not be converted","value":0}
,{"name":"E_NumberTimeOrderErrors","description":"Number of records not in ascending time order","value":0}
,{"name":"E_NumberNewAttributesNotAllowed","description":"Number of new attributes not allowed","value":0}
,{"name":"E_NumberNewAttributes","description":"Number of new attributes created","value":0}
,{"name":"E_NumberNewAttributesRecycled","description":"Number of new attributes recycled into existing space","value":0}
,{"name":"E_NumberByFields","description":"Number of 'by' fields within the model","value":5}
,{"name":"E_NumberOverFields","description":"Number of 'over' fields within the model","value":0}
,{"name":"E_NumberExcludedFrequentInvocations","description":"The number of times 'ExcludeFrequent' has been invoked by the model","value":0}
,{"name":"E_NumberSamplesOutsideLatencyWindow","description":"The number of samples received outside the latency window","value":0}
,{"name":"E_NumberMemoryLimitModelCreationFailures","description":"The number of model creation failures from being over memory limit","value":0}
,{"name":"E_NumberPrunedItems","description":"The number of old people or attributes pruned from the models","value":0}
]
[2021-01-19T15:40:43,747][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [Main.cc@284] Ml anomaly detector job exiting
[2021-01-19T15:40:43,747][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CBaseTokenListDataTyper.cc@54] Type=1-1 bar has landed 5
[2021-01-19T15:40:43,747][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job] [autodetect/511601] [CBaseTokenListDataTyper.cc@54] Type=2-9 foo has finished 10
[2021-01-19T15:40:43,748][TRACE][o.e.x.m.j.p.a.o.AutodetectStateProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Persisting job state document
[2021-01-19T15:40:43,759][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: refresh index .ml-anomalies-ml-snapshots-upgrade-job
[2021-01-19T15:40:43,767][TRACE][o.e.x.m.j.p.a.o.AutodetectStateProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Persisting job state document
[2021-01-19T15:40:43,771][DEBUG][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Quantiles queued for renormalization
[2021-01-19T15:40:43,773][TRACE][o.e.x.m.j.p.a.o.AutoDetectResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] Parsed ModelSizeStats: 74072 / 6 / 0 / 3 / 0 / ok
[2021-01-19T15:40:43,773][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] Persisting model size stats, for size 74072
[2021-01-19T15:40:43,773][TRACE][o.e.x.m.j.p.JobResultsPersister] [v6.8.14-1] [ml-snapshots-upgrade-job] ES API CALL: to index .ml-anomalies-.write-ml-snapshots-upgrade-job with ID [ml-snapshots-upgrade-job_model_size_stats_1611070843000]
[2021-01-19T15:40:43,775][TRACE][o.e.x.m.j.p.BatchedDocumentsIterator] [v6.8.14-1] ES API CALL: search index .ml-anomalies-ml-snapshots-upgrade-job
[2021-01-19T15:40:43,776][INFO ][o.e.x.m.p.AbstractNativeProcess] [v6.8.14-1] [ml-snapshots-upgrade-job] State output finished
[2021-01-19T15:40:43,777][DEBUG][o.e.x.m.p.AbstractNativeProcess] [v6.8.14-1] [ml-snapshots-upgrade-job] autodetect process exited
```

Logs from its snapshot trying to be loaded in 7.11.0
```
[2021-01-19T15:43:48,338][DEBUG][o.e.x.m.a.TransportGetModelSnapshotsAction] [v6.8.14-1] Get model snapshots for job ml-snapshots-upgrade-job snapshot ID null. from = 0, size = 100 start = 'null', end='null', sort=null descending=true
[2021-01-19T15:43:48,342][TRACE][o.e.x.m.j.p.JobResultsProvider] [v6.8.14-1] ES API CALL: search all model snapshots from index .ml-anomalies-ml-snapshots-upgrade-job sort ascending timestamp with filter after sort from 0 size 100
[2021-01-19T15:43:48,409][DEBUG][o.e.c.c.PublicationTransportHandler] [v6.8.14-1] received diff cluster state version [836] with uuid [nTXTjsYvSvCcHz0pGDZn9g], diff size [735]
[2021-01-19T15:43:48,424][INFO ][o.e.x.m.j.s.u.SnapshotUpgradeTaskExecutor] [v6.8.14-1] [ml-snapshots-upgrade-job] [1611070843] starting to execute task
[2021-01-19T15:43:48,448][DEBUG][o.e.x.m.j.p.JobResultsProvider] [v6.8.14-1] Found 0 hits for [[.ml-anomalies-ml-snapshots-upgrade-job]]
[2021-01-19T15:43:48,453][DEBUG][o.e.x.m.j.p.a.NativeAutodetectProcessFactory] [v6.8.14-1] [ml-snapshots-upgrade-job] Periodic operations staggered by 3084 seconds
[2021-01-19T15:43:48,454][DEBUG][o.e.x.m.j.p.a.NativeAutodetectProcessFactory] [v6.8.14-1] FieldConfig:
detector.0.clause = mean(value) partitionfield=series categorizationfield=text
detector.1.clause = count by mlcategory categorizationfield=text

[2021-01-19T15:43:48,454][INFO ][o.e.x.m.j.p.a.NativeAutodetectProcessFactory] [v6.8.14-1] Restoring quantiles for job 'ml-snapshots-upgrade-job'
[2021-01-19T15:43:48,455][DEBUG][o.e.x.m.p.NativeController] [v6.8.14-1] Command [2]: starting process with command [./autodetect, --jobid=ml-snapshots-upgrade-job, --bucketspan=3600, --lengthEncodedInput, --maxAnomalyRecords=500, --timefield=time, --persistInterval=13884, --maxQuantileInterval=24684, --config=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/config5778086115213841041.json, --limitconfig=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/limitconfig15639931706478400851.conf, --fieldconfig=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/fieldconfig14036308730085344193.conf, --quantilesState=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/ml-snapshots-upgrade-job_quantiles_4510667225169242506502.json, --deleteStateFiles, --logPipe=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job-1611070843_log_514256, --input=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job-1611070843_input_514256, --inputIsPipe, --output=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job-1611070843_output_514256, --outputIsPipe, --restore=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job-1611070843_restore_514256, --restoreIsPipe, --persist=/dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job-1611070843_persist_514256, --persistIsPipe, --namedPipeConnectTimeout=10]
[2021-01-19T15:43:48,763][DEBUG][o.e.x.m.p.NativeController] [v6.8.14-1] ML controller successfully executed command [2]: [Process './autodetect' started]
[2021-01-19T15:43:48,763][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [controller/514429] [CDetachedProcessSpawner.cc@291] Spawned './autodetect' with PID 517976
[2021-01-19T15:43:48,764][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [controller/514429] [CResponseJsonWriter.cc@38] Wrote controller response - id: 2 success: true reason: Process './autodetect' started
[2021-01-19T15:43:48,785][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CLogger.cc@333] Logger is logging to named pipe /dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/autodetect_ml-snapshots-upgrade-job-1611070843_log_514256
[2021-01-19T15:43:48,786][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [Main.cc@158] autodetect (64 bit): Version 7.11.0-SNAPSHOT (Build 9a819d67d56a96) Copyright (c) 2021 Elasticsearch BV
[2021-01-19T15:43:48,786][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CProcessPriority_Linux.cc@33] Successfully increased OOM killer adjustment via /proc/self/oom_score_adj
[2021-01-19T15:43:48,786][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CSystemCallFilter_Linux.cc@141] Seccomp BPF filters available
[2021-01-19T15:43:48,786][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CSystemCallFilter_Linux.cc@167] Seccomp BPF installed
[2021-01-19T15:43:48,848][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CFieldConfig.cc@192] Reading config file /dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/x-pack/qa/rolling-upgrade/build/testclusters/v6.8.14-1/tmp/fieldconfig14036308730085344193.conf
[2021-01-19T15:43:48,849][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CAnomalyJobConfig.cc@211] Parsing anomaly job config
[2021-01-19T15:43:48,849][INFO ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CResourceMonitor.cc@77] Setting model memory limit to 1024 MB
[2021-01-19T15:43:48,849][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CLengthEncodedInputParser.cc@48] Length encoded input parser input is not connected to stdin
[2021-01-19T15:43:48,849][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CHierarchicalResultsNormalizer.cc@290] 0 influencer bucket normalizers, 0 influencer normalizers, 1 partition normalizers, 1 person normalizers and 2 leaf normalizers restored from JSON stream
[2021-01-19T15:43:48,849][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [Main.cc@326] Applying the categorizer for anomaly detection
[2021-01-19T15:43:48,853][TRACE][o.e.x.m.j.p.StateStreamer] [v6.8.14-1] ES API CALL: get ID ml-snapshots-upgrade-job_model_state_1611070843#1 from index .ml-state*
[2021-01-19T15:43:48,862][TRACE][o.e.x.m.j.p.StateStreamer] [v6.8.14-1] ES API CALL: get ID ml-snapshots-upgrade-job_categorizer_state#1 from index .ml-state*
[2021-01-19T15:43:48,862][INFO ][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CAnomalyJob.cc@895] Processing is already complete to time 1491037200
[2021-01-19T15:43:48,863][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CAnomalyDetector.cc@115] CAnomalyDetector(): count by count for '', first time = 0, bucketLength = 3600, m_LastBucketEndTime = 0
[2021-01-19T15:43:48,863][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CAnomalyJob.cc@1045] Restoring state for detector with key '0==individual count/1/0//count////'
[2021-01-19T15:43:48,864][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CAnomalyDetector.cc@115] CAnomalyDetector(): count by mlcategory for '', first time = 0, bucketLength = 3600, m_LastBucketEndTime = 0
[2021-01-19T15:43:48,864][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CAnomalyJob.cc@1045] Restoring state for detector with key '1==individual rare count/0/0//mlcategory////'
[2021-01-19T15:43:48,864][ERROR][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CPersistUtils.h@1006] SubLevel mismatch in restore at b
[2021-01-19T15:43:48,865][ERROR][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CTimeSeriesModel.cc@1345] Failed to restore CONTROLLER_OLD_TAG, got 1
[2021-01-19T15:43:48,866][DEBUG][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [ml-snapshots-upgrade-job-1611070843] [autodetect/517976] [CStateDecompressor.cc@169] Explicit end-of-stream marker found in document with index 1
[2021-01-19T15:43:48,867][TRACE][o.e.x.m.j.p.StateStreamer] [v6.8.14-1] ES API CALL: get ID ml-snapshots-upgrade-job_categorizer_state#2 from index .ml-state*
[2021-01-19T15:43:48,875][DEBUG][o.e.x.m.j.p.a.JobModelSnapshotUpgrader] [v6.8.14-1] [ml-snapshots-upgrade-job] [1611070843] waiting for flush [1]
[2021-01-19T15:43:48,897][INFO ][o.e.x.m.p.AbstractNativeProcess] [v6.8.14-1] [ml-snapshots-upgrade-job] State output finished
[2021-01-19T15:43:48,897][ERROR][o.e.x.m.p.AbstractNativeProcess] [v6.8.14-1] [ml-snapshots-upgrade-job] autodetect process stopped unexpectedly: SubLevel mismatch in restore at b
Failed to restore CONTROLLER_OLD_TAG, got 1
Fatal error: 'si_signo 11, si_code: 1, si_errno: 0, address: 0x7ffb03edaf3e, library: /dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/distribution/archives/linux-tar/build/install/elasticsearch-7.11.0-SNAPSHOT/modules/x-pack-ml/platform/linux-x86_64/bin/../lib/libMlMaths.so, base: 0x7ffb038a2000, normalized address: 0x638f3e', version: 7.11.0-SNAPSHOT (build 9a819d67d56a96)

[2021-01-19T15:43:48,897][WARN ][o.e.x.m.j.p.a.o.JobSnapshotUpgraderResultProcessor] [v6.8.14-1] [ml-snapshots-upgrade-job] [1611070843] some model snapshot upgrade results not processed due to the termination of autodetect
[2021-01-19T15:43:48,902][DEBUG][o.e.c.c.PublicationTransportHandler] [v6.8.14-1] received diff cluster state version [837] with uuid [2zX1TOf8R8mrhQtrHLpEZA], diff size [1120]
[2021-01-19T15:43:48,914][ERROR][o.e.x.m.p.l.CppLogMessageHandler] [v6.8.14-1] [controller/514429] [CDetachedProcessSpawner.cc@184] Child process with PID 517976 was terminated by signal 11
[2021-01-19T15:43:48,932][DEBUG][o.e.c.c.PublicationTransportHandler] [v6.8.14-1] received diff cluster state version [838] with uuid [1Ruj4_BfS5SKs0ugOfQ0vQ], diff size [676]
[2021-01-19T15:43:48,956][WARN ][r.suppressed             ] [v6.8.14-1] path: /_ml/anomaly_detectors/ml-snapshots-upgrade-job/model_snapshots/1611070843/_upgrade, params: {snapshot_id=1611070843, job_id=ml-snapshots-upgrade-job, wait_for_completion=true}
org.elasticsearch.transport.RemoteTransportException: [v6.8.14-0][127.0.0.1:42843][cluster:admin/xpack/ml/job/model_snapshots/upgrade]
Caused by: org.elasticsearch.ElasticsearchException: Unexpected state [failed] while waiting for to be assigned to a node; recorded reason [[ml-snapshots-upgrade-job] autodetect process stopped unexpectedly: SubLevel mismatch in restore at b
Failed to restore CONTROLLER_OLD_TAG, got 1
Fatal error: 'si_signo 11, si_code: 1, si_errno: 0, address: 0x7ffb03edaf3e, library: /dev/shm/elastic+elasticsearch+7.11+matrix-java-periodic/ES_RUNTIME_JAVA/zulu11/nodes/general-purpose/distribution/archives/linux-tar/build/install/elasticsearch-7.11.0-SNAPSHOT/modules/x-pack-ml/platform/linux-x86_64/bin/../lib/libMlMaths.so, base: 0x7ffb038a2000, normalized address: 0x638f3e', version: 7.11.0-SNAPSHOT (build 9a819d67d56a96)
]
	at org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper.serverError(ExceptionsHelper.java:52) ~[?:?]
	at org.elasticsearch.xpack.ml.job.snapshot.upgrader.SnapshotUpgradePredicate.test(SnapshotUpgradePredicate.java:74) ~[?:?]
	at org.elasticsearch.xpack.ml.job.snapshot.upgrader.SnapshotUpgradePredicate.test(SnapshotUpgradePredicate.java:22) ~[?:?]
	at org.elasticsearch.persistent.PersistentTasksService.lambda$waitForPersistentTaskCondition$1(PersistentTasksService.java:153) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.cluster.ClusterStateObserver$ObserverClusterStateListener.clusterChanged(ClusterStateObserver.java:191) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.cluster.service.ClusterApplierService.callClusterStateListener(ClusterApplierService.java:526) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.cluster.service.ClusterApplierService.callClusterStateListeners(ClusterApplierService.java:517) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.cluster.service.ClusterApplierService.applyChanges(ClusterApplierService.java:484) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.cluster.service.ClusterApplierService.runTask(ClusterApplierService.java:418) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.cluster.service.ClusterApplierService.access$000(ClusterApplierService.java:68) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.cluster.service.ClusterApplierService$UpdateTask.run(ClusterApplierService.java:162) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:684) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:252) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:215) ~[elasticsearch-7.11.0-SNAPSHOT.jar:7.11.0-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

--- a/x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/MlJobSnapshotUpgradeIT.java
+++ b/x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/MlJobSnapshotUpgradeIT.java
@@ -88,7 +88,6 @@ public class MlJobSnapshotUpgradeIT extends AbstractUpgradeTestCase {
      * index mappings when it is assigned to an upgraded node even if no other ML endpoint is called after the upgrade
      */
     public void testSnapshotUpgrader() throws Exception {
-        assumeTrue("Mute due: https://github.com/elastic/elasticsearch/issues/67728", UPGRADE_FROM_VERSION.onOrAfter(Version.V_7_0_0));
         hlrc = new HLRC(client()).machineLearning();
         Request adjustLoggingLevels = new Request("PUT", "/_cluster/settings");
         adjustLoggingLevels.setJsonEntity(

